components/blocks/Features.tsx

```diff
@@ -1,24 +1,30 @@
-import { Actions } from './Actions'
-import { Container } from './Container'
-import { Prism } from '../styles/Prism'
-import { tinaField } from 'tinacms/dist/react'
-import DocsRichText from '../styles/DocsRichText'
-import styled from 'styled-components'
+import { Actions } from './Actions';
+import { Container } from './Container';
+import { Prism } from '../styles/Prism';
+import { tinaField } from 'tinacms/dist/react';
+import DocsRichText from '../styles/DocsRichText';
+import styled from 'styled-components';
+import playImage from '../../public/img/playButton.png';
 
 export function FeatureBlock({ data, index }) {
-  const isReversed = index % 2 === 1
+  const isReversed = index % 2 === 1;
+
+const handlePlayClick = () => {
+  const url = "https://youtube.com/tinacms"
+  window.open(url, '_blank');
+}
```

Comment:
why not use an `<a>` instead of a button if it just opens a link

Reply:
fixed